webpack.config.js

```diff
@@ -261,8 +248,14 @@ const common = {
   },
 
   resolve: {
-    modules: [paths.src, 'node_modules'],
-    extensions: ['.js', '.jsx', '.json', '.ts', '.tsx']
+    modules: [paths.distTsc, 'node_modules'],
+    extensions: ['.js', '.jsx', '.json', '.ts', '.tsx'],
+    extensionAlias: {
+      '.js': ['.js', '.ts'],
+      '.jsx': ['.jsx', '.tsx'],
+      '.cjs': ['.cjs', '.cts'],
+      '.mjs': ['.mjs', '.mts']
+    }
```

Comment:
fixes error in webpack when using .js imports inside TS files during `npm start`